production-deployment-using-tiup.md

```diff
@@ -161,7 +161,7 @@ TiUP 支持管理多个 TiDB 集群，该命令会输出当前通过 TiUP cluste
 Starting /home/tidb/.tiup/components/cluster/v1.0.0/cluster list
```

Comment:
Change `/home/tidb/.tiup/components/cluster/v1.0.0/cluster list` to `/home/tidb/.tiup/components/cluster/v1.3.0/cluster list`

Reply:
```suggestion
Starting /home/tidb/.tiup/components/cluster/v1.3.0/cluster list
```